lava-slave/Dockerfile

```diff
@@ -22,6 +22,13 @@ RUN rm /etc/apt/sources.list.d/testing.list
 RUN apt-get -y install ser2net
 COPY ser2net.conf /etc
 
+# ser2net > 3.2 is only availlable from sid
+RUN echo "deb http://deb.debian.org/debian/ sid main" >> /etc/apt/sources.list.d/sid.list
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install ser2net
+RUN rm /etc/apt/sources.list.d/sid.list
+RUN apt-get update
+
```

Comment:
Adding 'sid' here is also going to affect any of the packages that get installed after this though, right?  Like the lava-cli deps?

Reply:
I remove it and apt-get update just after using it